lib/security/security-defaults.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  enabled : false,
+  secret : 'keyboard_cat',
```

Comment:
For a default, I would set this to `null` or `undefined`, and throw an error at initialization of `security.enabled is true but `security.secret` is falsey to force users of this feature to provide an explicit secret.